pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "rbyte"
-version = "0.27.0"
+version = "0.27.1"
```

Comment:
the changes are limited to files that are effectively examples and aren't part of the package, so no bump necessary